src/main/java/com/camacho/simpleannotation/exceptions/ClassGenerationException.java

```diff
@@ -1,8 +1,8 @@
-package com.example.simpleannotation.exceptions;
+package com.camacho.simpleannotation.exceptions;
 
 public class ClassGenerationException extends RuntimeException{
```

Comment:
add space RuntimeException {

Reply:
Done. Revised in [pull request 8](https://github.com/OscarCamacho/simpleannotation/pull/8)